### PR TITLE
Merging changes from master/Stabilization for 2017.2.1.3 release

### PR DESCRIPTION
**Pull request by keveleigh**
Overview
---
This PR contains changes that were made after #1791.

### CODE REVIEW

File: Assets/MixedRealityToolkit-Examples/SharingWithUNET/Scripts/NetworkDiscoveryWithAnchors.cs.meta [hunk 1]
```diff
@@ -1,8 +1,9 @@
 fileFormatVersion: 2
-guid: 567e2a1ea65534040ac33441ec4bc34b
+guid: 63970e8e6c149264787d003d8583d1fd
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit-Examples/SharingWithUNET/Scripts/NetworkDiscoveryWithAnchors.cs.meta]: Not sure all these GUID changes are good.
Did these get changed in the master branch?
Could see people getting missing script references on their prefabs and in their scenes.

Comment (keveleigh) [reply Assets/MixedRealityToolkit-Examples/SharingWithUNET/Scripts/NetworkDiscoveryWithAnchors.cs.meta]: Yeah, that's weird. I'll dig in.

**Comment by StephenHodgson** [comment]
Is this a straight merge from master into the dev branch?

File: Assets/MixedRealityToolkit-Examples/SharingWithUNET/Scripts/NetworkDiscoveryWithAnchors.cs.meta (hunk 1 above)
Comment (keveleigh) [reply Assets/MixedRealityToolkit-Examples/SharingWithUNET/Scripts/NetworkDiscoveryWithAnchors.cs.meta]: Cool, the meta files have all been reverted. Not sure what changed them.

**Comment by keveleigh** [comment]
Yep, it's a merge + merge conflict resolution due to renamed files.

**Comment by StephenHodgson** [comment]
I'm surprised that even worked, esp bc of the namespace changes.

**Comment by keveleigh** [comment]
Yeah, I had to do quite a bit of manual conflict resolution! Surprisingly a few found their way on their own, but not many.

**Pull request merged**